circuits/bid/src/correctness.rs

```diff
@@ -147,4 +148,62 @@ mod tests {
         .is_err());
         Ok(())
     }
+
+    #[test]
+    fn test_correctness_with_message() -> Result<(), Error> {
+        use dusk_blindbid::Bid;
+        use dusk_pki::{PublicSpendKey, SecretSpendKey};
+        use dusk_poseidon::sponge;
+        use rand::Rng;
+
+        let mut rng = rand::thread_rng();
+        let secret = JubJubScalar::random(&mut rand::thread_rng());
+        let secret_k = BlsScalar::random(&mut rand::thread_rng());
+        let psk = PublicSpendKey::from(SecretSpendKey::new(
+            JubJubScalar::one(),
+            -JubJubScalar::one(),
+        ));
+        let value: u64 =
+            (&mut rand::thread_rng()).gen_range(V_RAW_MIN..V_RAW_MAX);
+
+        let bid = Bid::new(
+            Message::new(&mut rng, &secret, &psk, value),
+            sponge::sponge::sponge_hash(&[secret_k]),
+            psk.gen_stealth_address(&secret),
+            u64::MAX,
+            u64::MAX,
+        );
+
+        let (value, blinder) =
+            bid.decrypt_data(&secret, &psk).expect("decryption error");
+
+        let mut circuit = BidCorrectnessCircuit {
+            commitment: JubJubAffine::from(bid.commitment()),
+            value: value,
+            blinder: blinder,
+        };
+
+        // Generate Composer & Public Parameters
+        let pub_params = unsafe {
```

Comment:
Because it's just a single test..

But I'll do so. Remember that this was probably the first circuit to be created. So it does not follow the standards but because they were created later probably xD